test/testautomation_subsystems.c

```diff
@@ -173,16 +173,16 @@ static int SDLCALL subsystems_dependRefCountWithExtraInit(void *arg)
                         "Check result from SDL_WasInit(SDL_INIT_VIDEO | SDL_INIT_AUDIO | SDL_INIT_JOYSTICK | SDL_INIT_EVENTS)");
 
     /* Init EVENTS explicitly, +1 ref count. */
-    SDL_InitSubSystem(SDL_INIT_EVENTS);
+    SDLTest_AssertCheck(SDL_InitSubSystem(SDL_INIT_EVENTS), "SDL_InitSubSystem failure");
```

Comment:
I'd split this to multiple lines. One line to do the SDL call, and one line to test.

Reply:
Something like this?

```cpp
if (!SDL_InitSubSystem(SDL_INIT_VIDEO | SDL_INIT_AUDIO | SDL_INIT_JOYSTICK)) {
    SDLTest_AssertCheck(false, "SDL_InitSubSystem failure: %s", SDL_GetError());
}
```

?

Reply:
something like
```suggestion
    bresult = SDL_InitSubSystem(SDL_INIT_EVENTS);
    SDLTest_AssertCheck(bresult, "SDL_InitSubSystem returned %d, expected 1", bresult);
```
Ideally, the test code should contain no conditionals.